### ISSUE TITLE
[🐛 Bug]: No transparency on rounded window corners

### ISSUE THREAD

**Issue by jonrandy** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#827]
### What went wrong?

Update to 1.1, launched assistant, the rounded corners no longer look correct - no transparency
![image](https://user-images.githubusercontent.com/1510194/207380811-d91c52a0-615b-4c9f-b13e-6a109a614703.png)

### What did you expect to happen?

Corners should look nice as before

### Steps to reproduce the bug

Just open the app

### App Version

v1.1.0

### OS Version

Ubuntu 20.04.4 LTS

### Pre-submission Checklist

- [X] I have checked [the issue tracker](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues?q=is%3Aissue+label%3A%22%F0%9F%90%9B+bug%22) and didn't find any duplicate issue
- [X] I have checked the [Frequently Asked Questions (FAQs)](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Frequently-Asked-Questions-(FAQ)) and can confirm that there's no fixes or workarounds for this issue
- [X] This issue only covers one particular bug or issue

**Comment by im-coder-lg** [comment]
@sumeshir26 didn't you fix this? I'm surprised to see that thing again!

What was the fix given before? I bet it was outdated with new changes in the modules.

**Comment by sumeshir26** [comment]
> @sumeshir26 didn't you fix this? I'm surprised to see that thing again!
> 
> What was the fix given before? I bet it was outdated with new changes in the modules.

Not sure whats happening over here.

**Comment by im-coder-lg** [comment]
Similar/same as #357. @sumeshir26 you did make a PR on the fix but closed it. Perhaps it is the same issue?

**Comment by sumeshir26** [comment]
> Similar/same as #357. @sumeshir26 you did make a PR on the fix but closed it. Perhaps it is the same issue?

Yes, cuz the fix couldnt be tested

**Comment by im-coder-lg** [comment]
Well, I can test it, so can you make another PR when free? Let's see if we can close this issue soon!

**Comment by sumeshir26** [comment]
> Well, I can test it, so can you make another PR when free? Let's see if we can close this issue soon!

I dont think so, I dont have time at all for coding these days!

**Comment by Melvin-Abraham** [comment]
Hi @jonrandy, which format are you using? Is it Debian (`deb`) format or `snap` package?

**Comment by jonrandy** [comment]
It's the snap

**Comment by jonrandy** [comment]
Any news?

**Comment by sumeshir26** [comment]
> Any news?

@jonrandy No, since the author of this project hasnt been active on this repo lateley.
